src/style.css

```diff
@@ -1,3 +1,70 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --blue-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, serif;
```

Comment:
Consider adding more appropriate fallback fonts for `Roboto` to ensure better typography across different systems. For example, you could use `font-family: Roboto, Arial, sans-serif;`.